.flake8rc

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 79
```

Comment:
It is the default value. Why we need to specify it?

Reply:
There was no [such issue](https://github.com/muffinmad/anakin-language-server/actions/runs/6393375330/job/17354296944) on my computer.
And all because I have a local config with a different value:
```
# ~/.config/pycodestyle

[pycodestyle]
max-line-length = 110
```
So I think it is better to define this option as explicit.

P.S. How about increasing this value to at least 89? :)

Reply:
Well, I'm not sure I like the `Makefile` either :) But let's keep it for now. Thanks for your work!